src/schema/types/awsS3.graphql

```diff
@@ -0,0 +1,13 @@
+type S3Payload {
+  signedRequest: String!
+  url: String!
+}
+
+input SignS3Input {
+  filename: String!
+  filetype: String!
+}
+
+extend type Mutation {
+  signS3(input: SignS3Input!): S3Payload!
```

Comment:
I think `signS3` is not descriptive enough. What do you think of `createSignedFileUploadURL`? The input and return types could be `SignedFileUploadURLInput` and `SignedFileUploadURL`, respectively.
Also, the fact that we're using S3 is an implementation detail and shouldn't be in the mutation's name.

Reply:
Good idea